File: 진우/6weeks/2566. 최댓값/최댓값.py
arr = [list(map(int, input().split())) for _ in range(9)]

mx = -1
first = 0
second = 0

for i in range(9):
    for j in range(9):
       if arr[i][j] > mx:
           mx = arr[i][j]
           first = i + 1
           second = j + 1

print(mx)
print(first, second)

File: 진우/6weeks/2566. 최댓값/최댓값.cc
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int main() {
    int arr[9][9];
    for (int i = 0; i < 9; i++) {
        for (int j = 0; j < 9; j++) {
            cin >> arr[i][j];
        }
    }

    int mx = -1;

    int first = 0;
    int second = 0;

    for (int i = 0; i < 9; i++) {
        for (int j = 0; j < 9; j++) {
            if (arr[i][j] > mx) {
                mx = arr[i][j];
                first = i;
                second = j;
            }
        }
    }

    cout << mx << "\n";
    cout << first + 1 << " " << second + 1 << "\n";

    return 0;
}
